utils/TetrisUtils.tsx

```diff
@@ -4,26 +4,39 @@ import styles from "styles/components/Tetris.module.css";
  * @brief: fieldToJsxElement: Convert a field of coords to a JSX element for
  * render
  * @param  {number[][]}     field  Field to render
+ * @param  {boolean}        over   Whether the game is over
  * @param  {boolean}        grid   Toggle for the game grid
+ * @param  {boolean}        queue  Whether the field is for the queue panel
  * @return {JSX.Element[]}         JSX element of field
  */
 export const fieldToJsxElement = (
   field: number[][],
+  over: boolean = false,
   grid: boolean = false,
   queue: boolean = false
 ): JSX.Element[] => {
   const retField: JSX.Element[] = [];
 
   field.forEach((col, colIndex) => {
-    const rows = col.map((row, rowIndex) => (
-      <div
-        className={`
-          ${styles.row} ${styles[grid ? `row-wgrid-${row}` : `row-${row}`]}
-          ${styles[queue ? "row-queue" : ""]}
-        `}
-        key={`row-${rowIndex}`}
-      />
-    ));
+    const rows = col.map((row, rowIndex) => {
+      let rowStyle;
```

Comment:
I think it would a better practice if we assign a safe default value to variables before actually working with them. Of course it's not super important here but would be nice to have